tests/core/editable/removemultiplelist.js

```diff
@@ -0,0 +1,116 @@
+/* bender-ckeditor-plugins: list,toolbar,table,format */
+
+( function() {
+	'use strict';
+
+	bender.editors = {
+		classic: {},
+		inline: {
+			creator: 'inline'
+		}
+	};
+
+	var tests = {
+		'test remove full multiple list': function( editor, bot ) {
```

Comment:
I would be more precise in all of those tests names. For example, in this case, instead of `full multiple list` I would use `multiple fully selected list` or something similar.

And for an overall convention, I would go with: `mutiple fully selected lists are removed when delete key is pressed`.

Please correct all tests names according to the above convention.